src/FakeItEasy/FakeItEasy.nuspec

```diff
@@ -13,10 +13,41 @@
     <licenseUrl>https://github.com/FakeItEasy/FakeItEasy/blob/master/License.txt</licenseUrl>
     <copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</copyright>
     <tags>TDD unittesting mocks mocking fakes faking stubs stubbing spy spies doubles isolation substitutes substitution</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.0">
+        <dependency id="Newtonsoft.Json" version="9.0.1" />

```

Comment:
I think so too. If we drop this feature in .NET Core (at least for now), it probably won't be missed by many people. It's certainly not one of the core features.